system-addon/lib/TelemetryFeed.jsm

```diff
@@ -78,25 +79,30 @@ this.TelemetryFeed = class TelemetryFeed {
     // introduce, rather than doing the correct by complicated thing.  It may
     // well be worth reexamining this hypothesis after we have more experience
     // with the data.
-    let absBrowserOpenTabStart =
-      perfService.getMostRecentAbsMarkStartByName("browser-open-newtab-start");
+    let absBrowserOpenTabStart;
+    try {
+      absBrowserOpenTabStart = perfService.getMostRecentAbsMarkStartByName("browser-open-newtab-start");
+    } catch (e) {
+      // Just use undefined so it doesn't get sent to the server
+    }
+
+    // If we're missing either starting timestamps, treat it as an unexpected
+    // session; otherwise, assume it's the usual behavior.
+    const triggerType = absBrowserOpenTabStart === undefined ||
+      absVisChangeTime === undefined ? "unexpected" : "menu_plus_or_keyboard";
 
```

Comment:
Please update the system-addon version of  `data_events.md` and `data_dictionary.md` and `pings.js` to include the `unexpected` value.